lib/reline/ansi.rb

```diff
@@ -60,8 +61,36 @@ def self.set_default_key_bindings(config)
       config.add_default_key_binding_by_keymap(:emacs, key, func)
     end
   end
+  
```

Comment:
These 10 lines of code already exist below, so now it is in two places.  Did you mean to remove it from below?